identifiers/country-in/lok_sabha.csv

```diff
@@ -549,9 +549,11 @@ ocd-division/country:in/territory:dl/cd:south_delhi,Delhi Lok Sabha constituency
 ocd-division/country:in/territory:dl/cd:west_delhi,Delhi Lok Sabha constituency West Delhi,,
 ocd-division/country:in/territory:dn/cd:dadra_and_nagar_haveli,Dadra and Nagar Haveli Lok Sabha constituency Dadra and Nagar Haveli (ST),,2020-01-25
 ocd-division/country:in/territory:jk/cd:anantnag,Jammu and Kashmir Lok Sabha constituency Anantnag,2019-10-31,
+ocd-division/country:in/territory:jk/cd:anantnag-rajouri,Jammu and Kashmir Lok Sabha constituency anantnag-rajouri,,

```

Comment:
Can you add the date from when the renaming was applied ?

Reply:
Sure, please see the updates.